### PR TITLE
restructure install section and clarify raspberry pi install options

### PR DESCRIPTION
**Pull request by indirectlylit**
* refactors 'install' section according to https://github.com/learningequality/kolibri-docs/issues/45 and https://github.com/learningequality/learningequality.org/pull/479
* adds stub section for mac
* adds initial language clarifying the relationship between the raspberry pi OS image and the tutorial
* delete dead/draft/hidden content

### CODE REVIEW

**Comment by indirectlylit** [comment]
Note to all: if this looks like a reasonable base to work off of, please merge in and submit additional PRs to that so I don't become a bottleneck

**Review by benjaoming — APPROVED** [review]

**Comment by benjaoming** [comment]
It's about going through steps and learning them, so I think it's a good idea to retain the "tutorial" as a deliberate part of the format, visible to both documentation maintainers and readers.

File: docs/install/index.rst
Comment (radinamatic) [inline docs/install/index.rst:15]: This step (our _setup wizard_) logically goes **after** the installation, so I don't think this is the best place.

**Comment by radinamatic** [comment]
I rather liked *Tutorials* being a separate section, so not sure why the decision to eliminate them...? Maintaining that section would not be in conflict with the rest of your proposed additions here.

Current | Proposed
------------ | -------------
 | 

I will approve and merge, but will move **Initial setup** back at the end in the upcoming update, and would like to reinstate tutorials as a separate sections.

**Comment by indirectlylit** [comment]
My motivation for removing the 'tutorials' section was following the guidance in https://github.com/learningequality/kolibri-docs/issues/45 and similar work in the developer docs here: https://github.com/learningequality/kolibri/pull/4536 - namely, that flat is better than nested.

It allowed the top-level "Install" page to be formatted thusly:

image

However, if you guys prefer a sub-section for tutorials that's fine with me.

Regarding the re-ordering of the items, I put then in order of generality and relevance, rather than sequence of use. In other words, everyone needs the system requirements and setup wizard docs, but the others are more context specific.

However, again I defer to you on this.

**Comment by radinamatic** [comment]
I understand what you are saying. 

I like the top-level "Install" page formatting, and will try to implement that for the rest of pages according to what we discussed in #45 (although that might be a bigger refactor, to be polished in the subsequent updates after the 0.13 release), and I'll see if it's doable to make it into this update.

**Review by radinamatic — APPROVED** [review]
Thank you! 👍

**Comment by benjaoming** [comment]
Agreeing about flat structure... The number of clicks required to reach something is great to keep down, having good and well-placed cross references is important.

I was concerned about eliminating "Tutorial" as a word and sticking to the format. Because install instructions, to me, are something different and that an "instruction" framing will erode the tutorial-oriented text over time.

Installation instructions = as simple and minimal as possible
Tutorials = with learning purposes

The other thing about tutorials is that they are written with their own scoped specifications: "In this tutorial, we show you how you can do X on platform Y".

I like tutorials, I think we should have more :)

**Pull request merged**
